surefire-provider/src/main/java/org/arquillian/smart/testing/surefire/provider/TestStrategyApplier.java

```diff
@@ -9,15 +9,37 @@
 
 class TestStrategyApplier {
 
+    static final String FILTERING = "filtering";
```

Comment:
Have you considered `selecting`? In most of the documents and studies related to "smart testing", there is mainly used the word "selection" in the context of running some subset of test classes.